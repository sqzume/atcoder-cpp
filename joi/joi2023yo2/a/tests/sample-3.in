10
440894064 101089692 556439322 34369336 98417847 216265879 623843484 554560874 247445405 718003331
