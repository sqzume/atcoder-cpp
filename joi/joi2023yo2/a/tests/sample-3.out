406524728
616913639
522069986
683633995
619585484
501737452
589474148
520191538
470557926
683633995
